sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/logs/_exporter.py

```diff
@@ -65,6 +67,7 @@ def shutdown(self) -> None:
     def _log_to_envelope(self, log_data: LogData) -> TelemetryItem:
         if not log_data:
             return None
+        envelope = None
```

Comment:
Why is this added here?

Reply:
It was taken from your branch but ill remove it.